source/faq.txt

```diff
@@ -115,3 +115,10 @@ If you see slow operation warnings, check CPU, memory, and network usage on the
 source and destination clusters.  If the clusters are underprovisioned for your
 needs, consider upgrading the cluster hardware.
 
+Should I stop a migration that encounters error or failure logs?
+----------------------------------------------------------------
+
+No, error and failure logs are non-fatal errors and do not signal that
+you need to terminate ``mongosync`` early. These logs do not indicate
```

Comment:
Update terminate -> stop per the [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#t)

```suggestion
you need to stop ``mongosync`` early. These logs do not indicate
```

Reply:
Done!